lib/types/operators/Modulo.cpp

```diff
@@ -0,0 +1,211 @@
+#include "../../../includes/types/JsAny.hpp"
```

Comment:
for all this file you should check this links to catch all errors if needed

https://en.cppreference.com/w/cpp/numeric/math/fmod
https://en.cppreference.com/w/cpp/numeric/fenv/FE_exceptions

add missing catch for when std::stod failed